src/mame/commodore/kim1.cpp

```diff
@@ -58,15 +73,17 @@ Paste test:
 
 
 TODO:
+- Get console working on screen (works with pty)
 - LEDs should be dark at startup (RS key to activate)
-- add TTY support
+- Make console toggle a DIP switch?
```

Comment:
What is it physically?  If it isn’t a DIP switch, it should not be a DIP switch in MAME.  Did you choose `IPT_KEYBOARD` just so it could easily be enabled/disabled in the Keyboard Selection menu when you want to use the terminal keyboard?

Speaking of which, is the reason the terminal isn’t “working” because you have multiple devices with keyboard inputs in the system and the terminal keyboard isn’t enabled?  MAME only enables the first keyboard in the system by default so you aren’t typing on all the keyboards at once. (go to main menu, Input Settings, Keyboard Selection to enable/disable keyboards).